src/chapters/booleans-and-conditionals/conditionals.rst

```diff
@@ -0,0 +1,233 @@
+Conditionals
+============
+
+.. index:: ! conditional
+
+A the beginning of this chapter, we motivated our exploration of booleans in JavaScript by observing that having a formal notion of truth and falsehood would allow us to write code that executes conditionally. In other words, we want to be able to write code that only executes when a given condition is ``true``.
+
+Again, here is our motivating example:
+
+.. admonition:: Example
+
+   Consider a banking application that can remind you when a bill is due. The application will notify the user that a bill is due soon, but *only if* the bill has not already been paid.
```

Comment:
Earlier rephrasing suggestion: "Consider a banking application that reminds you to pay a bill on time. The app sends you a notification _only if_ you miss the due date."